debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -689,9 +710,20 @@ private static int validateTableExcludeList(Configuration config, Field field, V
         return Collections.unmodifiableMap(snapshotSelectOverridesByTable);
     }
 
+    private static int validateSchemaBlacklist(Configuration config, Field field, Field.ValidationOutput problems) {
+        String whitelist = Configuration.getFallbackStringProperty(config, SCHEMA_INCLUDE_LIST, SCHEMA_WHITELIST);
```

Comment:
Why is it using the fallback approach here? Shouldn't it _only_ validate the legacy one?

Reply:
@gunnarmorling I didn't quite follow your question, `SCHEMA_WHITELIST` is the deprecated option and hence calls the fallback method to log a warning. That is the main idea. Isn't?

Reply:
We need this at the moment for blacklist properties. But agreed, it's a bit weird!
More specifically we use that to trigger the log message (on a different field) when both options (`*.exclude.list and `*.whitelist`) were set.
Although this works I was thinking about defining another (internal) field that does the validation of all the other fields in one place. The question here is: how to properly handle multi-field validation properly (other options?). This is definitely the correct way with the current implementation.